docs/MFScope.md

```diff
@@ -24,6 +24,27 @@ All users and appIds are unique:
 - Two users with same user part cannot exist (if "123~A" exists, "123~B" cannot exist).
 - Two users with same appId cannot exist (if "123#netflix" exists, "456#netflix" cannot exist).
 
+Example:
+
+| Incoming UserId | Existing UserId | 	Result 	  |
+|-----------------| --------------- |   ------ 	  | 
+| 123			  | 123				| Not accepted|
+| 123			  | 123~A			| Not accepted|
+| 123			  | 123~A#appId		| Not accepted|
+| 123			  | 123#appId		| Not accepted|
```

Comment:
I'm not sure why the formatting looks weird over here.You can view the exact formatted table text in view file.Here is the screenshot of the table
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/121901963/235768319-c91c1bda-0f35-47f4-8a23-7d4ee15420b0.png">